src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -353,10 +353,27 @@ private static void downloadJavaFXZip(String osarch, Path substratePath, Project
         Logger.logDebug("Process zip javafx done");
     }
 
+    /**
+     * Download the zip file from the specified URL into the path provided by destination, and unpack it there
+     * Note the conventions for the url file (must contain .zip) and the destination (must not contain .zip,
+     * as it points to the ultimate directory that is the result of the unzip).
+     * @param urlZip the URL location of the zip file, e.g. https://download2.gluonhq.com/substrate/clibs/foo.zip
+     * @param destination the location where the zip file should be downloaded and unpacked,
+     *                    e.g. graalvm/libs/clibraries/foo
+     *                    The zip file will be installed in graalvm/libs/clibraries/foo.zip
+     *                    The contents of this zip file will be installed in graalvm/libs/clibraries/foo
+     * @throws IOException
+     */
+    public static void downloadZip(String urlZip, Path destination) throws IOException {
```

Comment:
why `public`?

Reply:
As this is an API that other components can use (e.g. AbstractTargetConfiguration).